apps/emqx_utils/test/emqx_utils_maps_tests.erl

```diff
@@ -129,3 +131,28 @@ key_comparer_test() ->
             #{}
         ])
     ).
+
+map_indent_unindent_test_() ->
+    M = #{a => 1, b => 2},
+    [
+        ?_assertEqual(
+            #{a => 1, c => #{b => 2}},
+            indent(c, [b], M)
+        ),
+        ?_assertEqual(
+            M,
+            unindent(c, indent(c, [b], M))
+        ),
+        ?_assertEqual(
+            #{a => 1, b => #{b => 2}},
+            indent(b, [b], M)
+        ),
+        ?_assertEqual(
+            M,
+            unindent(b, #{a => 1, b => #{b => 2}})
+        ),
+        ?_assertEqual(
+            #{a => 2},
+            unindent(b, #{a => 1, b => #{a => 2}})
+        )
+    ].
```

Comment:
Suggestion: test indenting with empty list and more than 1 indentation key, unindent absent key, indent absent key.